lib/app/modules/alarmRing/controllers/alarm_ring_controller.dart

```diff
@@ -52,7 +52,7 @@ class AlarmControlController extends GetxController {
     return latestAlarm;
   }
 
-  getNextAlarm() async {
+  getAlarm() async {
```

Comment:
Why was this function renamed?

Reply:
sorry, i replaced next unfortunately. i will solved all the problem make a fresh commit. Thanks